src/dapp/libexec/dapp/dapp-verify-contract

```diff
@@ -88,41 +92,87 @@ else
   optimized=0
 fi
 
-params=(
+if  [[ $1 = "--multiple" ]]; then
```

Comment:
Can you please rework this to use the unified option parsing code in `src/dapp/libexec/dapp/dapp`. You need to parse the `--multiple` flag using the infrastructure in place there and then conditionally set an environment variable (e.g. `DAPP_VERIFY_CONTRACT_MULTIPLE`) if the `--multiple` flag is present.

You can then branch on the envrionment variable here.

Reply:
Done. Let me know if it's ok! 
I'll send a pr on `src/dapp/libexec/dapp` with the env variable asap :slightly_smiling_face: